views/index.erb

```diff
@@ -47,13 +47,79 @@
 
 <!-- Add your Javascript to this script tag -->
 <script language="javascript">
-  console.log("I am Javascript!");
+	console.log("I am Javascript!");
 
-  var profile_values = document.getElementsByClassName('profile-value');
-  var total_words = 0;
+	var profile_values = document.getElementsByClassName('profile-value');
+	var total_words = 0;
+	var word_list = "";
+	for (var i in profile_values) {
+		if (profile_values[i].textContent != undefined) {
+			word_list += (profile_values[i].textContent + " ");
+		}
+	}
+	var filtered_list = word_list.split(" ").filter(function(val) { return val !== ""; });
```

Comment:
This is good and works, the downside is it's a little more memory intensive and we're concatenating large strings only to split up again. We'll show you an approach where you can use regexp to look for the number of occurences and it's a little easier on the machine.